build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1280,6 +1292,11 @@ message ExecuteRequest {
   // The server will have a default policy if this is not provided.
   // This may be applied to both the ActionResult and the associated blobs.
   ResultsCachePolicy results_cache_policy = 8;
+
+  // A list of acceptable encodings to used for inlined data.  Must be
+  // `IDENTITY` (or unspecified), or one or more of the compressors
+  // supported by the server.
+  Compressor.Value compressor = 9;
```

Comment:
This should be a repeated field, named "acceptable_compressors" for consistency.